lib/test.js

```diff
@@ -106,8 +107,11 @@ Test.prototype.test = function (name, opts, cb) {
     });
 };
 
-Test.prototype.comment = function (msg) {
+Test.prototype.comment = function () {
     var that = this;
+    var keys = Object.keys(arguments)
+    var msg = keys.length > 1 ? format.apply(null, arguments) : arguments[keys[0]];
```

Comment:
Use case would be, e.g., `console.log`-style debugging where you want to log object contents. With the change as it stands you can do `t.cmt('%j', obj)`.  Formatting everything, you could do `t.cmt(obj)`.

Doesn't sound like 5 fewer characters is worth the risk of breaking older code tho.